applications/rag/README.md

```diff
@@ -164,22 +152,22 @@ This step generates the vector embeddings for your input dataset. Currently, the
     * When the last cell says the job has succeeded (eg: `Job 'raysubmit_APungAw6TyB55qxk' succeeded`), the vector embeddings have been generated and we can launch the frontend chat interface.
     * Ray may take several minutes to create the runtime environment. During this time, the job will appear to be missing (e.g. `Status message: Job has not started yet`).
 
-### Launch the Frontend Chat Interface
+### Launch the frontend chat interface
 
-#### With IAP Disabled
+#### With IAP disabled
 1. Setup port forwarding for the frontend: `kubectl port-forward service/rag-frontend -n $NAMESPACE 8080:8080 &`
 
 2. Go to `localhost:8080` in a browser & start chatting! This will fetch context related to your prompt from the vector embeddings in the `pgvector-instance`, augment the original prompt with the context & query the inference model (`mistral-7b`) with the augmented prompt.
 
-#### With IAP Enabled
+#### With IAP enabled
 1. Verify that IAP is enabled on [Google Cloud Platform (GCP) IAP](https://console.cloud.google.com/security/iap)(make sure you are logged in) for your application. If you encounter any errors, try re-enabling IAP.
 2. From *Google Cloud Platform IAP*, check if the target user has role `IAP-secured Web App User`. This role is necessary to access the application through IAP.
```

Comment:
unrelated to your change but since you're updating the README:

Can you add a hyperlink to Google Cloud Platform IAP (https://console.cloud.google.com/security/iap)) to 2. ?
and also: Grant access to additional users via GCP IAP with role `IAP-secured Web App User`?

Reply:
don't bother re-running gcbrun if you fix this

Reply:
Fixed.